e2e/vertical-full-page-map.spec.ts

```diff
@@ -17,27 +17,41 @@ test.describe('full page map test suite', () => {
     await page.getByPlaceholder('Search for locations').fill('virginia');
     await page.getByPlaceholder('Search for locations').press('Enter');
     await page.getByRole('button', { name: 'Result number 5' }).click();
-    const locator = page.locator('#js-answersVerticalResults div').nth(134);
-    await expect(locator).toHaveClass(/pinFocused/);
+    const results = page.locator('#js-answersVerticalResults div');
+    const count = await results.count();
+    let hasPinFocused = false;
+
+    for (var i = 0; i < count; i++) {
```

Comment:
can we use a stream function (probably `some`) here instead of a for loop? or could we even directly check for a match for a locator on the `yxt-Card--pinFocused` class?